runtime/testy/namedSensors.csv

```diff
@@ -0,0 +1,3 @@
+two,2
```

Comment:
On a separate note:
* Maybe use the more general term "peripherals" instead of "sensors"
* Why is this file in version control? It will eventually be user-writable (though doesn't seem that way yet). This means that pretty soon you'll need to figure out a cleaner scheme for switching between a real file and a dummy file for testing.
* What happens in the case of one hibike board with multiple attached sensors (i.e. two limit switches)? Last year's sensor mapping scheme allowed assigning a unique name to each switch, meaning that this created two entries in the mappings table.